pkgs/development/compilers/bs-platform/build-bs-platform.nix

```diff
@@ -57,8 +57,7 @@ stdenv.mkDerivation rec {
     cp jscomp/others/*.ml jscomp/others/*.mli jscomp/others/*.cm* $out/lib/ocaml
     cp jscomp/stdlib-406/*.ml jscomp/stdlib-406/*.mli jscomp/stdlib-406/*.cm* $out/lib/ocaml
     cp bsconfig.json package.json $out
-    ln -s $out/bsb $out/bin/bsb
-    ln -s $out/bsc $out/bin/bsc
-    ln -s $out/bsrefmt $out/bin/bsrefmt
+    ln -s $out/bsb $out/bsc $out/bsrefmt $out/bin
+    ln -s $out/bsb $out/bsc $out/bsrefmt $out/lib
```

Comment:
can you explain this change? Thanks!

Reply:
Sure, line 60 is just a rewriting of the removed lines into a single command, and line 61 makes the same symlinks in `$out/lib` - this is necessary for reason-language-server to work: when you have a project directory, e.g. from `bsb -init my-reason-project -theme basic-reason`, and then you `cd` into the directory, reason-language-server tries to call `bsb` not from `PATH`, but from `./node_modules/.bin/bsb` (I guess this is some node convention), which is a symlink to `${bs-platform}/lib/bsb`. Without this change, the symlink is dangling.

Reply:
good stuff, thanks